SoCCo/algorithms/climate.py

```diff
@@ -1,10 +1,10 @@
-### This folder contains the climate and social functions for the coupled social
-### and climate model
+### This folder contains the climate functions for the coupled social
```

Comment:
Documentation about a module should use `"""` instead of comments so that they get picked up by ipython's `?`.